.codeclimate.yml

```diff
@@ -20,6 +20,7 @@ engines:
     enabled: true
   rubocop:
     enabled: true
+    channel: rubocop-0-51
```

Comment:
Looks like the latest version is now 0.52:
https://github.com/codeclimate/codeclimate-rubocop/branches/all?utf8=%E2%9C%93&query=channel%2Frubocop